features/FEATURE_LWIP/lwip-interface/ppp_lwip.cpp

```diff
@@ -18,6 +18,14 @@
 #include "platform/FileHandle.h"
 #include "platform/mbed_poll.h"
 #include "events/EventQueue.h"
+#if defined(FEATURE_COMMON_PAL)
+#include "mbed_trace.h"
+#define TRACE_GROUP "LPPP"
+#else
+#define tr_debug(...) (void(0)) //dummies if feature common pal is not added
```

Comment:
When mbed-trace will become part of platform/ or something, we can change the code. This part of the code is internal to us.
There are issues with LWIP tracing,  for example huge memory consumption and overlapping of serial communication.